packages/rocketchat-katex/lib/katex

```diff
@@ -0,0 +1 @@
+../../../node_modules/katex/dist/
```

Comment:
Hm i've tried, the problem is that it I need to add katex fonts as assets so I can't just import them and it seems that meteor restricts the files I can add as assets to be inside the package directory.
I've tried using `Npm.depends()` at first instead of adding katex as a dependency in the global package.json, but it seems that meteor runs the `Package.onUse()` callback before installing the npms of the package..
Do you have an idea how to get it done?